### PR TITLE
chore: disable codeflow for pull requests

### PR DESCRIPTION
**Pull request by sean-perkins**
Issue number: N/A

---------

## What is the current behavior?

Team isn't using the integration. 

## What is the new behavior?

- Disables the Codeflow integration on pull requests

## Does this introduce a breaking change?

- [ ] Yes
- [x] No

### CODE REVIEW

File: .stackblitz/codeflow.json [hunk 1]
```diff
@@ -4,7 +4,7 @@
       "enabled": false
```
Comment (liamdebeasi) [inline .stackblitz/codeflow.json:4]: If we're going to disable everything, should we remove the file entirely?

**Review by liamdebeasi — APPROVED** [review]
One non blocking question

**Review by thetaPC — APPROVED** [review]
LGTM

File: .stackblitz/codeflow.json (hunk 1 above)
Comment (sean-perkins) [reply .stackblitz/codeflow.json:4]: Their documentation mentions disabling the bot via the configuration file or through admin permissions for the organization: https://developer.stackblitz.com/codeflow/integrating-codeflowapp-bot#disabling-codeflowapp-bot

My thoughts were disabling it via the configuration and then follow-up with deleting the config file to make sure it is in fact disabled for the repository. 

I believe Codeflow is closed sourced, so I'm unable to see how it detects the enabled status (if it reads from `main`, stashes to an external service, etc.).

**Review by liamdebeasi — APPROVED** [review]

**Pull request merged**